Makefile

```diff
@@ -45,14 +46,15 @@ lint:
 
 build:
 	@echo "+ $@"
-	@go build -tags "${DOCKER_BUILDTAGS}" -v ${GO_LDFLAGS} ./...
+	@go build -v ${GO_LDFLAGS} ./...
 
 test:
 	@echo "+ $@"
-	@go test -tags "${DOCKER_BUILDTAGS}" ./...
+	@go test ./...
 
 binaries: ${PREFIX}/bin/continuity
 	@echo "+ $@"
+	@go build -o $< ${GO_LDFLAGS} ./cmd/continuity
```

Comment:
This line doesn't need to be here, `${PREFIX}/bin/continuity` already causes this to be built.